serial_tests/test_serial.py

```diff
@@ -0,0 +1,136 @@
+import queue
+import random
+import socket
+import threading
+
+from dataclasses import dataclass
+from typing import Tuple
+
+import pytest
+from RCTComms.transport import (RCTAbstractTransport, RCTSerialTransport)
+
+NUM_TRIALS = 128
+TARGET_IP = '127.0.0.1'
+TARGET_PORT = 'COM2'
+
+@dataclass
+class TransportPair:
+    client: RCTAbstractTransport
+    server: RCTAbstractTransport
+
+def transport_open(transport: RCTAbstractTransport):
+    """
+    Attempts to open the transport
+
+    Args:
+        transport (RCTAbstractTransport): Transport to open
+    """
+    while True:
+        try:
+            transport.open()
+            return
+        except ConnectionError:
+            continue
+
+def server_connection_handler(connection, id):
```

Comment:
missing function docstring
unused arguments

Reply:
Resolved with 662a21c
Removed function in test_serial since these functions were for testing TCP server